apps/integration/src/app/integrations/lookup-in-modal/modal-lookup.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
```

Comment:
Is there a test for this component? The "integrations" app is a place to put unit tests for complex component interactions. If the intent is to have an example that can be interacted with manually, we should put this in the playground app.

Reply:
This is in the integration app to support the respective e2e test. I can add a unit test for this component.

Reply:
I didn't realize we had an "integration" E2E application. Thanks for creating the unit test! 👍